examples/chip-tool/commands/clusters/CustomArgument.h

```diff
@@ -230,9 +232,30 @@ class CustomArgument
 
     CHIP_ERROR Parse(const char * label, const char * json)
     {
-        Json::Reader reader;
         Json::Value value;
-        reader.parse(json, value);
+        constexpr const char kHexNumPrefix[] = "0x";
+        constexpr size_t kHexNumPrefixLen    = ArraySize(kHexNumPrefix) - 1;
+        if (strncmp(json, kPayloadHexPrefix, kPayloadHexPrefixLen) == 0 ||
```

Comment:
Curious if we could use std::string instead for the comparisons

Reply:
We could, but we'd need to make a copy of the json, etc.  And even then, the way I found to test for "std::string starts with something" is kinda obtuse (involving `rfind(prefix, 0) == 0`)...  So in terms of readability, it wasn't much of a win, if at all, sadly.